src/features/context-menu/components/ContextMenuComponent.js

```diff
@@ -408,4 +417,22 @@ function ensureFocus(el) {
       setRange(focusEl, { start: 100000, end: 100000 });
     }
   }
+}
+
+function getTableContainerScroll(node) {
+  const tableContainer = domClosest(node, '.tjs-container');
+
+  if (!tableContainer) {

```

Comment:
Is this possible? I thought the table components are always enclosed by `.tjs-container`.

Reply:
Not really, just wanted to be extra safe. 🔒

Reply:
I think this is technically possible with `createPortal`, so maybe it's good to leave it as it is.